deps/nghttp2.version

```diff
@@ -1,5 +1,6 @@
+# -*- makefile -*-
```

Comment:
```suggestion
# vim:ft=make:
```

Reply:
Would 
```suggestion
# vim:ft=make: -*- makefile -*-
```
work for vim?  It would for Emacs and GitHub: https://github.com/JuliaLang/julia/blob/f06f2141c12b1da1764f3adebabe1b5166138ebe/deps/nghttp2.version#L1-L6

Reply:
Yea, but it was mostly a joke :)